.env.testing

```diff
@@ -1,3 +1,3 @@
 BASE_URL="https://react-shopping-cart-67954.firebaseapp.com/"
 USERNAME=testingUserName
-PASSWORD=testingPassword
+PASSWORD=testingPassword
```

Comment:
Not an issue here, but it is good practice to hide any sensible data (like passwords) in environment variables or secrets in GitHub. Take a look at the secrets docs in GitHub and chuck the Password in there :)

Reply:
I didn't consider that as sensitive as it was just to show what was going on i have another env which is in the gitignore  and which is put in the github secrets.